### PR TITLE
Fixing robocup bash functions for new folder stucture

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Work in progress

### CODE REVIEW

File: setup/tue.bash_functions [hunk 1]
```diff
@@ -719,7 +729,7 @@ function tue-robocup-update
     cd ~/.tue
     git pull --ff-only
 
-    tue-robocup-install-package picaso_4d_systems
+    tue-robocup-install-package picaso_4d_systems #WHY????
```
Comment (LoyVanBeek) [inline setup/tue.bash_functions]: Indeed, why? 
This line is added in https://github.com/tue-robotics/tue-env/commit/2fa40d7a3b818fac127963a5ce801462495495aa by @svddries and I think it was for a touch screen we had back then. Relatively sure this is OK for removal again.

Comment (reinzor) [inline setup/tue.bash_functions]: Yes, can be removed .. We are not using this anymore and there is no use for that line being there

**Comment by MatthijsBurgh** [comment]
@LoyVanBeek or @reinzor, Please a final review

**Comment by LoyVanBeek** [comment]
I don't see anything to disagree with :-)

**Pull request merged**